mobile/src/navigation/routes.ts

```diff
@@ -0,0 +1,12 @@
+import { List } from 'immutable';
+
+export const routeComponents = (navigationState: any): List<string> => {
```

Comment:
Isn't there a type for it?

Reply:
@santee 
There is, but it doesn't describe the full structure.

Reply:
extend it, maybe? 'any' is just a time bomb.